grails-app/i18n/messages.properties

```diff
@@ -2580,6 +2581,7 @@ react.stockMovement.confirmPreviousPage.continue.label=Continue (lose unsaved wo
 react.stockMovement.message.confirmSubmit.label=Confirm submit
 react.stockMovement.confirmSubmit.message=Please confirm you are ready to submit your request. Once submitted, you cannot edit the request.
 react.stockMovement.confirmPickRefresh.message=This button will redo the autopick on all items that have not been previously edited. Are you sure you want to continue?
+react.stockMovement.orderNumber.label=PO number
```

Comment:
Same here. Please call this Order Number.

Reply:
We should really try to consolidate these i.e. `react.stockMovement` and `stockMovement` should be merged into `stockMovement`. But we can save that for another day.